.github/workflows/benchmark.yml

```
@@ -135,7 +137,7 @@ jobs:
         working-directory: ./benchmarks/tools
 
       - name: Benchmark
-        run: dotnet run -c Release -b ${{ matrix.benchmark }}
+        run: dotnet run -c Release -f net6.0 -b ${{ matrix.benchmark }}
```

Revert benchmark build changes, this will be captured in another work item
```suggestion
        run: dotnet run -c Release -b ${{ matrix.benchmark }}
```